x-pack/plugins/security_solution/public/common/components/risk_score/risk_score_disabled/translations.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
```

Comment:
You have these titles defined here too, maybe we can consolidate? Not necessary for this PR but maybe in cleanup. `x-pack/plugins/security_solution/public/overview/components/entity_analytics/common/translations.ts`